astroquery/simbad/core.py

```diff
@@ -642,6 +681,11 @@ def query_objects(self, object_names, *, wildcard=False, criteria=None,
             When set to `True` the method returns the HTTP request parameters without
             querying SIMBAD. The ADQL string is in the 'QUERY' key of the payload.
             Defaults to `False`.
+        async_job: bool, optional
+            When set to `True`, the query will be executed in asynchronous mode. This is
+            better for very long queries, as it prevents transient failures to abort the
+            query execution.
+            Defaults to `False`.
         cache : Deprecated since 0.4.8. The cache is now automatically emptied at the
```

Comment:
unrelated, but I noticed it now: the deprecated keywords can actually be cleaned up from the docstrings

Reply:
Done! :slightly_smiling_face: